walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/Utils.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps;
+
+import java.util.Set;
+
+public class Utils {
```

Comment:
nit: final to disable inheritance + private constructor to disable construction of your Utils class.